config/3.3/aiManager/templates/gobal-image-pull-secret.yaml

```diff
@@ -0,0 +1,125 @@
+{{- if .Values.internalbuild }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: gobal-image-pull-secret
+  annotations:
+    argocd.argoproj.io/sync-wave: "40"
+  namespace: {{.Values.spec.aiManager.namespace}}
+spec:
+  template:
+    spec:
+      containers:
+        - name: config
+          image: quay.io/openshift/origin-cli:latest
+          imagePullPolicy: IfNotPresent
+          resources:
+            requests:
+              memory: "500Mi"
+              cpu: "500m"
+            limits:
+              memory: "500Mi"
+              cpu: "500m"
+          command:
+            - /bin/sh
+            - -c
+            - |
+              set -eo pipefail
+              set -x
+
+              ARTIFACTORY_EMAIL={{.Values.ARTIFACTORY_EMAIL}}
```

Comment:
Can we use an array where each element represents one registry including the repo, username, and password. By doing so, we can just iterate over the array in this job and we do not have to know each registry details beforehand which gives us the flexibility of adding arbitrary repo info when needed.

Also, suggest to use camelCase, instead of UPPER_CASE for helm params to keep consistency.

Reply:
Will commit a patch to change this